addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8646,8 +8661,10 @@ msgctxt "#19056"
 msgid "New timer"
 msgstr ""
 
+#. Label for a context menu entry to open timer settings dialog
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
 msgctxt "#19057"
-msgid "Edit timer"
+msgid "Edit settings"
```

Comment:
@da-anda

> "New timer" DOES show up in context menu of a timer

 Overlooked this. Sorry. Was tricked by the fact that we have "Add timer" (used in ctx menu of epg events in search and guide window) and "New timer" used in ctx menu of timers. I agree that "New timer" in ctx menu of timers should go.

> "Find similar" is ofc fine on EPG entries etc - just feels a bit odd on a timer. 

 Right. IMO it does not make sense to have "Find similar" in ctx menu of a timer as it implies to find what? A similar timer? Actually it tries to get an epg event from the timer and searches for a similar event then, or if the timer has no epg event, it searches for an epg event matching timer's title (which is strictly speaking nonsense as the timer's title can almost always freely choosen by the user).